OperatingSystem-Services/Platform-Linux/HostBasedFirewall/IPTables/README.md

```diff
@@ -1 +1,362 @@
+### IPTables Basic Syntax
+When using the `iptables` command, we will use the following general syntax; of course, this may vary depending on what you would like to do!
+
+#### Adding or Deleting Rules
+```sh
+$ iptables -t <TABLE> [-A|-I|-D] <Chain> <Rule>
+```
+
+First we use the `iptables` command, if you have the necessary programs installed - Most of the time it's `iptables` but you should google this yourself! - we should not run into any issues when running the command unless you have other host-based firewalls installed. Below is a set of sample output if `iptables` is ran without any arguments.
+```
+$ iptables
+iptables v1.8.9 (nf_tables): no command specified
+Try `iptables -h' or 'iptables --help' for more information.
+```
+> [!NOTE]
+> Notice how we can see (nf_tables), this is because we are not using the traditional iptables kernel API/interface, we are using the nftables interface. More details can be found online in articals such as ["iptables: The two variants and their relationship with nftables"](https://developers.redhat.com/blog/2020/08/18/iptables-the-two-variants-and-their-relationship-with-nftables#).
+
+Next we generally specify the table we would like to be working with using the `-t` flag. This is not always necessary since the *default* is the **filter** table which is the one will are often most concerned with.
+
+> [!NOTE]
+> The table name should be *all lowercase*
+
+
+After this we specify the action we would like to take, there are additional commands for creating and managing *chains* but this is outside of the *basics*. Below are three actions you will commonly use to manage the *rules* within a *chain*:
+* `-A`: Append, this will create a rule at the *end* or *tail* of the chain. This becomes the last rule checked in the chain.
+* `-I`: Insert, this will create a rule at the *beginning* or *head* of the chain. This becomes the first rule checked.
+  * `-I <Chain> <Num>`: We can optionally specify a location to insert the rule at.
+* `-D <Chain> <NUM>`: Delete, we specify an index within a chain of a rule to be removed
+
+We always need to specify the *chain* to preform the action on, you will need to know the *name* or *identifier* of the chain and this can be seen when we list the contents of a table. Most tables share names such as `INPUT`, `OUTPUT`, `PREROUTING`, etc.
+
+If we are not deleting a rule, we then include the necessary flags to define the rule we would like to add to our system.
+
+We can also *flush* all the rules contained within a *table* or *chain* using the following command.
+```
+$ iptables -t <table> -F <chain>
```

Comment:
A person reading this can make the guess that -t means table & -F means flush, but it may be beneficial to explicitly write down the what the flag means similiar to line 92

```suggestion
$ iptables -t <table> -F <chain>
- t: "table" specify table to be modified
- f: "flush" removes rules
```